scripts/step-3.sh

```diff
@@ -47,11 +47,11 @@ checkout_branch
 # Skip if only timestamping
 if [ -z "$DISABLE_SNAPSHOT" ]; then
     ./tuf snapshot -repository "$REPO"
-    ./tuf sign -repository "$REPO" -roles snapshot -key "${SNAPSHOT_KEY}"
+    ./tuf sign -repository "$REPO" -roles snapshot -key "${SNAPSHOT_KEY}" --add-old-type
```

Comment:
This change was present in #1130 and tested here https://github.com/sigstore/root-signing/pull/1137